alphagov/paas-product-pages
# Create a Hello world app

Hello world [NextJS(https://nextjs.org/)] app with 
- static page build
- [MDX](https://mdxjs.com/) to author content in Markdown, but have the flexibility to add components

This is the basic setup with necessary dependencies and updated README file.

How to review
-----

- clone project
- optional: install `node` if you don't have it already
- install dependencies with `npm install`
- run `npm run dev` to build the app
- go to `http://localhost:3000` to view the page
- Check you see **Hello world**
- optional: Edit `pages/index.mdx` and observe the change
